modules/javafx.graphics/src/main/java/com/sun/glass/ui/Accessible.java

```diff
@@ -162,36 +162,28 @@ private class GetAttribute implements PrivilegedAction<Object> {
 
     private GetAttribute getAttribute = new GetAttribute();
 
-    @SuppressWarnings("removal")
     public Object getAttribute(AccessibleAttribute attribute, Object... parameters) {
-        AccessControlContext acc = getAccessControlContext();
-        if (acc == null) return null;
-        return QuantumToolkit.runWithoutRenderLock(() -> {
-            getAttribute.attribute = attribute;
-            getAttribute.parameters = parameters;
-            return AccessController.doPrivileged(getAttribute, acc);
-        });
+        getAttribute.attribute = attribute;
+        getAttribute.parameters = parameters;
+        return QuantumToolkit.runWithoutRenderLock(getAttribute);
     }
 
-    private class ExecuteAction implements PrivilegedAction<Void> {
+    private class ExecuteAction implements Supplier<Void> {
         AccessibleAction action;
         Object[] parameters;
-        @Override public Void run() {
+        @Override public Void get() {
             eventHandler.executeAction(action, parameters);
             return null;
         }
     }
 
     private ExecuteAction executeAction = new ExecuteAction();
 
-    @SuppressWarnings("removal")
     public void executeAction(AccessibleAction action, Object... parameters) {
-        AccessControlContext acc = getAccessControlContext();
-        if (acc == null) return;
         QuantumToolkit.runWithoutRenderLock(() -> {
             executeAction.action = action;
             executeAction.parameters = parameters;
-            return AccessController.doPrivileged(executeAction, acc);
+            return executeAction;
```

Comment:
I just noticed that this has the same problem that `getAttribute` had, which I reported and you fixed. The return value isn't used (so we didn't get a fail fast), but I'm pretty sure that this will cause a problem in that `executeAction` will never be executed.

For consistency, you might consider changing it in the same way as you did for `getAttribute`.

Reply:
I have updated executeAction also. Looks like this call reaches from MacAccessible under certain conditions on macOS.

To instrument and verify, i ran different a11y tests in Hello toy(with VoiceOver) and found out that clicking on any Item in `HelloSimpleTableView` calls executeAction. So after the update also we receive this call.